### PR TITLE
CC-13273 Disaply Number in Zed documentation.

### PR DESCRIPTION
**Pull request by romanhavrylko**
## PR Description

Epic: https://spryker.atlassian.net/browse/CC-13273

Ticket for IG: https://spryker.atlassian.net/browse/CC-13287

## Checklist
- [ ] I agree with the Code Contribution License Agreement in CONTRIBUTING.md

### CODE REVIEW

**Comment by DmitryLymarenko** [comment]
JS adjustments were made at Gui module
I didn't see frontend building is mentioned

**Review by DmitryLymarenko — DISMISSED** [review]

File: _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md [hunk 1]
```diff
@@ -0,0 +1,624 @@
...
+```
+
+{% info_block warningBox “Verification” %}
+
+Having finished the entire integration, ensure that:
+
+*   Entries without a translation for a language with a configured fallback are translated into the fallback language.
+
+*   Translation cache is stored under the configured directory.
+
```
Comment (jeremyfourna) [inline _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md:181]: Remove the extra line between each point

File: _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+console twig:cache:warmer
+console twig:cache:warmer
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that:
+
+*   You can open the Back Office login page or any page which requires authentication.
+
```
Comment (jeremyfourna) [inline _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md:366]: Remove the extra line between each point

File: _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md [hunk 3]
```diff
@@ -0,0 +1,624 @@
...
+    }
+}
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that you’ve enabled the plugins:
+
+1.  In the Back Office, go to **Users** > **Users**.
+
```
Comment (jeremyfourna) [inline _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md:563]: Remove the extra line between each point

File: _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+console translator:clean-cache
+console translator:generate-cache
+```
+
+{% info_block warningBox "Verification" %}
+
+Ensure that the command:
+
+*   cleaned previous translation cache in the translation folder, which is `data/{YourStore}/cache/Zed/translation` by default.
+
```
Comment (jeremyfourna) [inline _includes/pbc/all/install-features/202212.0/install-the-spryker-core-back-office-feature.md:621]: Remove the extra line between each point

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
+---
+title: "HowTo: Add support of number formatting in Backoffice"
+description: Add support of numbers formatting in Backoffice UI.
+template: howto-guide-template
+last_updated: Aug 30, 2022
+---
+
+#{Title}
+
+{Overview}
```
Comment (jeremyfourna) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md:10]: Why the brackets are needed here?

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+| Spryker Core                           | {{page.version}} | [Spryker Core feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/spryker-core-feature-integration.html)                           |
+| Promotions & Discounts (Optional)      | {{page.version}} | [Promotions & Discounts feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/promotions-and-discounts-feature-integration.html)     |
+| Product Options (Optional)             | {{page.version}} | [Product Options feature walkthrough](/docs/scos/dev/feature-walkthroughs/{{page.version}}/product-options-feature-walkthrough.html)                           |
+| Product + Order Management (Optional)  | {{page.version}} | [Product + Order Management feature integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/product-order-management-feature-integration.html) |
+| Shipment (Optional)                    | {{page.version}} | [Shipment integration](/docs/scos/dev/feature-integration-guides/{{page.version}}/shipment-feature-integration.html)                                           |
+
+## {Step-by-step instructions}
```
Comment (jeremyfourna) [inline docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md:26]: Why the brackets are needed here?

**Review by jeremyfourna — CHANGES_REQUESTED** [review]
Fix small typos and structure

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md (hunk 6 above)
Comment (romanhavrylko) [reply docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md:26]: Removed.

File: docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md (hunk 5 above)
Comment (romanhavrylko) [reply docs/scos/dev/tutorials-and-howtos/howtos/feature-howtos/202212.0/howto-add-support-of-number-formatting-in-backoffice.md:10]: Removed.

**Review by jeremyfourna — APPROVED** [review]
OK for me

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**